src/auto_archiver/modules/generic_extractor/twitter.py

```diff
@@ -6,7 +6,7 @@
 
 from auto_archiver.core.metadata import Metadata, Media
 from auto_archiver.utils import UrlUtil
-from auto_archiver.archivers.archiver import Archiver
+from auto_archiver.base_modules.extractor import Extractor
```

Comment:
How about just `auto_archiver.base.extractor` ? Since these aren't actually modules per se, they are just mixins. In fact, you don't even *have* to subclass from them anymore if you don't want to. They are just there for convenience and typing.